README.md

```diff
@@ -402,6 +402,7 @@ A curated list of awesome Java frameworks, libraries and software.
 *Libraries that assist with the creation, evaluation or manipulation of graphical images.*
 
 * [Imgscalr](https://github.com/thebuzzmedia/imgscalr) - Simple and efficient hardware-accelerated image-scaling library implemented in pure Java 2D.
+* [Tess4J](https://github.com/nguyenq/tess4j) - A Java JNA wrapper for Tesseract OCR API. It provides optical character recognition (OCR) support for: TIFF (single and multipage), JPEG, GIF, PNG, BMP and PDF
```

Comment:
Remove `A Java` and `OCR` (both redundant). The second sentence describes Tesseract not the Wrapper. Suggestion for the description: `JNA wrapper for Tesseract OCR.`